composer.json

```diff
@@ -27,7 +27,9 @@
     "ext-grpc": "*",
     "firebase/php-jwt": "^6.3",
     "google/protobuf": "3.21.5",
-    "grpc/grpc": "1.42.0"
+    "grpc/grpc": "1.42.0",
+    "monolog/monolog": "^3.2",
+    "psr/simple-cache": "^3.0"
```

Comment:
so this means that our main SDK now has these new deps, correct?

Just food for thought - whether this might cause transitive dependency conflicts for existing apps.  We generally want to try to keep the dep list as minimal as possible.

It might be worth packaging the PSR implementation separately for this reason, but I have mixed feelings about that because I want it to be easily discoverable.

No change requested for now but just something to think about.  If we keep it like this for 1.0 then I think it would technically require a 2.0 release to separate them out.